### ISSUE TITLE
what's the learning rate schedule for training WaveRNN?

### ISSUE THREAD

**Issue by lin88lin8850** [fatchord/WaveRNN#116]
if I train wavenet from scratch with 'MOL' mode, the loss didn't decrease after 300k.

**Comment by fatchord** [comment]
I'd recommend using 'RAW' mode if you are new to the model. 'MOL' can sometimes take around a million steps to sound good whereas 'RAW' will sound good a lot faster. 8bit is fastest while 9bit takes a bit longer.

**Comment by lin88lin8850** [comment]
solved, closed
